### DISCUSSION TITLE
1.1.1 - Minor fixes

### DISCUSSION THREAD

**Discussion by XorTroll** [XorTroll/emuiibo#265] [General]
# Changelog

## emuiigen

- Dependencies were adjusted to properly support Java 8 as well

  - A consequence of this is that combo boxes are no longer searchable (for now)

<hr /><em>This discussion was created from the release <a href='https://github.com/XorTroll/emuiibo/releases/tag/1.1.1'>1.1.1 - Minor fixes</a>.</em>

**Comment by Blacknife420** [comment]
Hello are you going to keep the original one (emutool) going as I can't get this new one to work

**Reply by XorTroll** [reply]
Open an issue with your problem then (if it's not a duplicate)

**Comment by kekisan** [comment]
Hi, I was wondering if it works with 18.0.0 version

**Comment by Blacknife420** [comment]
Yea mate it dose I've tested

**Reply by kekisan** [reply]
thanks for your answer, I asked because I've seen many people struggle with the 18.0.0 version

**Reply by Alexdelreyy** [reply]
It's not working for me, help? :(

**Comment by Blacknife420** [comment]
Odd thay use a new computer side called emuigen or something like that maby
that might be ur problem idk how to help as mine worked fine after updating

**Comment by totalnewb** [comment]
How do i use emuiigen? There is no executable file to run.

**Reply by impeeza** [reply]
It IS A EXECUTABLE,  a Java executable, you run java executables with the java interpreter of your OS.

**Comment by instakills** [comment]
tengo problemas para ejecutar el archivo.jar alguna recomendacion? estuve probando varios metodos y nada

**Comment by benjamin9078-ctrl** [comment]
Hello, and thank you very much for emuiibo.
It is an amazing project and still widely used.

Currently, on recent Atmosphère versions (1.10+), the emuiibo overlay
is no longer compatible, which makes it impossible to select or activate
virtual amiibos through Tesla / Ultrahand.

The sysmodule still loads correctly, but without an updated overlay,
users cannot choose an amiibo, so emuiibo cannot be used in games.

Would it be possible to update the overlay to be compatible with
new Atmosphère and Ultrahand, or provide guidance on an alternative way
to select virtual amiibos?

Thank you very much for your work and time.
